test/cookbooks/test/recipes/extension.rb

```diff
@@ -1,28 +1,37 @@
 postgresql_repository 'install'
+# # Dokken images don't have all locales available so this is a workaround
+locale = case node['platform_family']
+         when /^(debian|ubuntu|fedora)$/
+           'C.UTF-8'
+         when 'rhel'
+           if node['hostname'].match?(/oracle/)
+             'C.UTF-8'
+           else
+             node['platform_version'].to_i < 8 ? 'en_GB.UTF-8' : 'C.UTF-8'
+           end
+         when 'oracle'
+           'C.UTF-8'
+         else
+           'en_US.UTF-8'
+         end
 
-# Dokken images don't have all locales available so this is a workaround
-locale = value_for_platform(
-  [:debian, :ubuntu, :fedora] => { default: 'C.UTF-8' },
-  centos: { default: 'en_GB.utf-8' },
-  default: 'en_US'
-)
-
+# postgresql_server_install 'package'
```

Comment:
Why is this commented out?

Reply:
I had entered that to test without the special options and forgot to remove it. I have started reading diffs before pushing, but sometimes forgetting